xbmc/GUIInfoManager.cpp

```diff
@@ -8201,6 +8228,7 @@ const infomap pvr[] =            {{ "isrecording",              PVR_IS_RECORDING
                                   { "backendrecordings",        PVR_BACKEND_RECORDINGS },
                                   { "backenddeletedrecordings", PVR_BACKEND_DELETED_RECORDINGS },
                                   { "backendnumber",            PVR_BACKEND_NUMBER },
+                                  { "backendinstancename",      PVR_BACKEND_INSTANCE_NAME },
```

Comment:
I cannot find an implementation for this new label!

Reply:
https://github.com/xbmc/xbmc/blob/77eacb8b9c665b7b5db0efa8892078213efec163/xbmc/guilib/guiinfo/GUIInfoLabels.h#L672

Reply:
That is not enough. You need to actually implement it in PVRGUIInfo.cpp, like you did with the other new labels/ints/bools.

Reply:
I have reviewed the PR and `PVR.BackendInstanceName` is never referred to in the skin or anywhere else.  As this InfoLabel is not required to achieve the desired functionality, I have removed it.

This PR was submitted 5 months ago and was on hold since then due to activities surrounding the v21 release.  Nobody is at fault for this delay, however, I am no longer able to remember my thought process regarding this particular InfoLabel.  Any further comment would be pure speculation.